cfme/base/ui.py

```diff
@@ -404,6 +405,15 @@ def step(self):
         self.view.navigation.select('Cloud Intel', 'RSS')
 
 
+@navigator.register(Server)
```

Comment:
Optional:
I would use short name like just `Timeline` against `Server` object.

Reply:
@digitronik this is to mirror the fact that `CloudIntel` is used at the start of views elsewhere, e.g. `CloudIntelReportsView` Just `Timelines` is also used elsewhere.

Reply:
best practice to use `short_name` but its totally optional :+1: